airflow/api_connexion/openapi/v1.yaml

```diff
@@ -843,6 +843,26 @@ paths:
         '404':
           $ref: '#/components/responses/NotFound'
 
+  /providers:
```

Comment:
Can you add this change to "Summary of changes"? http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow/latest/stable-rest-api-ref.html#section/Summary-of-Changes

Reply:
I added a new line in that summary table, let me know if its fine .